.gitignore

```diff
@@ -47,3 +47,5 @@ velocity.log*
 examples/**/Dependencies.toml
 load-tests/**/Dependencies.toml
 ballerina/Dependencies 2.toml
+$(pwd)
+native/src/main/java/io/ballerina/stdlib/io/nativeimpl/.CsvChannelUtils.java.icloud
```

Comment:
what are these files?

Reply:
one of them($(pwd)) is coming from vs code ballerina the file with .iCloud file automatically created when downloading the backups. I removed both and updated the file.